CompuCell3D/core/Demos/Models/cellsort_2_5_D/Simulation/cellsort_2D.xml

```diff
@@ -0,0 +1,70 @@
+<!--This demo shows how to use 2.5D setting. With 2.D enabled (<DimensionType>2.5D</DimensionType>) -->
+<!--the pixel copies are only in the xy plane. -->
+<!--If you comment this setting out you will see pixels scattering all over the lattice -->
+
+
+<CompuCell3D>
+    <Potts>
+        <Dimensions x="100" y="100" z="20"/>
+        <Anneal>10</Anneal>
+        <Steps>10000</Steps>
+        <Temperature>1000</Temperature>
+        <Flip2DimRatio>0.1</Flip2DimRatio>
+        <NeighborOrder>1</NeighborOrder>
+<!--        <MetropolisAlgorithm>BoundaryWalker</MetropolisAlgorithm>-->
+        <DimensionType>2.5D</DimensionType>
```

Comment:
Besides this line, is there anything else people should be aware of when creating/designing 2.5D simulations? It can go in the docs.